modules/order.py

```diff
@@ -881,8 +881,8 @@ def doArchiveActiveOrdersTask(self, timeStamp, cursor):
 
 		newCursor = query.getCursor()
 
-		if gotAtLeastOne and newCursor and newCursor.urlsafe() != cursor:
-			self.doArchiveActiveOrdersTask(timeStamp, newCursor.urlsafe())
+		if gotAtLeastOne and newCursor and newCursor != cursor:
```

Comment:
Couldn't this now be simplified into just
```python
if newCursor:
```
and gotAtLeastOne is over and can be removed?